schema/Hexapod.yaml

```diff
@@ -0,0 +1,56 @@
+$schema: http://json-schema.org/draft-07/schema#
+$id: https://github.com/lsst-ts/ts_hexapod/blob/master/schema/Hexapod.yaml
+# title must end with one or more spaces followed by the schema version, which must begin with "v"
+title: Hexapod v1
+description: Schema for Hexapod configuration files
+type: object
+properties:
+  x_max:
+    description: The max limit along the x-axis
+    type: number
+    default: 1
+  y_max:
+    description: The max limit along the y-axis
+    type: number
+    default: 1
+  z_max:
+    description: The max limit along the z-axis
+    type: number
+    default: 1
+  u_max:
+    description: The max limit along the u-axis (rot about x-axis)
+    type: number
+    default: 1
+  v_max:
+    description: The max limit along the v-axis (rot about y-axis)
+    type: number
+    default: 1
+  w_max:
+    description: The max limit along the w-axis (rot about z-axis)
+    type: number
+    default: 1
+
```

Comment:
Can you remove the extra lines at the end? leave only one blank lines, no spaces..

Reply:
Done